rclcpp/include/rclcpp/experimental/subscription_intra_process.hpp

```diff
@@ -102,6 +102,12 @@ class SubscriptionIntraProcess : public SubscriptionIntraProcessBase
 #endif
   }
 
+  ~SubscriptionIntraProcess()
+  {
+    rcl_ret_t ret = rcl_guard_condition_fini(&gc_);
+    (void)ret;
```

Comment:
if we keep the current code, we do not even need `ret`?

```suggestion
    (void) rcl_guard_condition_fini(&gc_);
```

Reply:
Writing as that is to avoid build warning 'not deal with return value of rcl_guard_condition_fini()'.  
Now I change code to output error log which also can avoid that build warning.